### DISCUSSION TITLE
bruh- anyone know some browsers I can use to unblock it on this crusty musty rusty ass school laptop?

### DISCUSSION THREAD

**Discussion by MisakeTheRat** [Prodigy-Hacking/ProdigyMathGameHacking#1246] [(A) Anything]
![Screenshot 2022-01-03 11 27 40 PM](https://user-images.githubusercontent.com/96935110/148154633-87f9850e-85bd-4268-82b2-9e8ca4995337.png)

**Comment by afkvido** [comment]
build phex from source

**Reply by MisakeTheRat** [reply]
wdym

**Reply by MisakeTheRat** [reply]
elaborate

**Reply by MisakeTheRat** [reply]
> build phex from source

Can you help me please?

**Reply by afkvido** [reply]
**bruh**

**Reply by afkvido** [reply]
ok

**Reply by afkvido** [reply]
if you don't know what that means

**Reply by afkvido** [reply]
just download the latest release through releases

**Comment by donaldli2020** [comment]
use a personal account
